roles/resource_manager/includes/build.yaml

```diff
@@ -4,24 +4,24 @@
     os_name: "{{ ansible_network_os }}"
     name: "{{ ansible_network_os }}_{{ res }}"
     state: gathered
-  register: result
+  register: resource_manager_result
```

Comment:
@KB-perByte  Is there any reason for this change ? is it also reserved ?

Reply:
@rohitthakur2590 these aren't the reserved ones these changes were added bcoz of the rule var-naming[no-role-prefix]
```
var-naming[no-role-prefix]: Variables names from within roles should use role_name_ as a prefix. (register: result)
roles/resource_manager/includes/build.yaml:2 Task/Handler: Run the platform facts module
```

Reply:
ref https://github.com/redhat-cop/network.base/actions/runs/5111475234/jobs/9188472559#step:4:63

This PR fixes 2 different lint failures that the content had! 
```
      var-naming[no-reserved]    basic   idiom                
      var-naming[no-role-prefix] basic   idiom  
```